src/js/initializers/skip-nav.js

```diff
@@ -8,12 +8,15 @@ whenDOMReady(function () {
   var elSkipnav = select('.skipnav')[ 0 ];
   var elMainContent = select('#main-content')[ 0 ];
 
-  dispatch(elSkipnav, 'click', function () {
-    elMainContent.setAttribute('tabindex', '0');
-  });
-
-  dispatch(elMainContent, 'blur', function () {
-    elMainContent.setAttribute('tabindex', '-1');
-  });
+  if (elSkipnav) {
+    dispatch(elSkipnav, 'click', function () {
```

Comment:
No idea. I haven't changed the events that were there before, just rearranged them. It's very possible, but I don't have enough experience with event types and our target behaviours to make that decision. :(